dec14_exercise.py

```diff
@@ -0,0 +1,60 @@
+ages = [
+29,
+19,
+17,
+26,
+32,
+16,
+21,
+
+]
+#4
+print(ages)
+#6
+print(ages[4])
+#7
+print(ages[-2])
+#8
+print(ages[2:6])
+#9
+print(ages[:3])
+#10
+if 17 in ages:
+    print(True)
+else:
+    print(False)
+#11
+if 42 in ages:
+    print(True)
+else:
+    print(False)
+
+#13
+ages[2]=18
+print(ages)
+
+#14
+temp=ages[4]
+ages[4]=ages[3]
+ages[3]=temp
+print(ages)
+
+#17
+ages.append(45)
+print(ages)
+
+#18
+ages.insert(0,32)
+print(ages)
+
+#19
+ages.insert(4,37)
+print(ages)
```

Comment:
That's just before the fifth value, not the sixth value:

---

![image](https://user-images.githubusercontent.com/16143771/146121512-505277b0-5553-41b2-a6ee-b60b38d49f8e.png)